contest/views.py

```diff
@@ -222,3 +222,11 @@ def reply(request):
                     % (request.user.username, replied_clarification.id))
             return redirect('contest:contest',contest)
     return redirect('contest:archive')
+
+def download(request):
+    what = request.POST.get('type')
+    if what == 'scoreboard':
```

Comment:
In my opinion, it doesn't need to check any permission since everyone can see scoreboard.